packages/dataviews/src/types.ts

```diff
@@ -3,21 +3,49 @@
  */
 import type { ReactNode } from 'react';
 
-interface Option {
-	value: any;
+export type SortDirection = 'asc' | 'desc';
+
+/**
+ * Type of view UI.
+ */
+export type ViewType = 'table' | 'grid' | 'list';
```

Comment:
I don't think we should limit the view type because we want to allow registration of views ultimately.

Reply:
I can remove it. I have just found it useful to have a type for what dataviews are available and passing a typed parameter down to the render functions of the fields. Because they typically need to render differently in specific view types.

Reply:
Changed it back to `string`.